### PR TITLE
Enikshay updates in the live version of reports

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz 

These are updates in the live version of reports which pass the QA. Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1914,6 +1914,7 @@ def _determine_couch_databases(couch_databases):
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'rntcp_pmdt_treatment_register.json'),
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'referral_report_v2.json'),
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'drug_voucher.json'),
+    os.path.join('custom', 'enikshay', 'ucr', 'reports', 'dnc_lab_summary.json'),
```
Comment (calellowitz) [inline settings.py]: why are you adding a report that is not in this PR? was it left out of a different one?

**Review by calellowitz — CHANGES_REQUESTED** [review]
report looks good. one question about the settings change

File: settings.py (hunk 1 above)
Comment (lwyszomi) [reply settings.py]: @calellowitz 
Was added to the repository several PR earlier (https://github.com/dimagi/commcare-hq/pull/18701). I forgot to add it to the settings.

Comment (calellowitz) [reply settings.py]: :+1:

Comment (calellowitz) [reply settings.py]: looks like this should be `dmc_lab_summary.json`

Comment (lwyszomi) [reply settings.py]: :+1:

**Review by calellowitz — APPROVED** [review]

**Pull request merged**